docs/docs/userguide/INSTALLATION.md

```diff
@@ -72,7 +72,7 @@ If you want to use a custom configuration file to handle more than one environme
 ```
 
 !!! help "How to run the product after installation?"    
-    When the installation process finishes successfully, you can find some detailed information about the infrastructure on your console including the endpoint url (`product_urls`/`load_balancer_hostname`) to open the product on your browser and more.      
+    When the installation process finishes successfully, you can find some detailed information about the infrastructure in the console, including the `<product_url>`/`<load_balancer_hostname>` endpoint URL to launch the product in the browser and more.      
```

Comment:
Please remove `/<load_balancer_hostname> endpoint URL` from this line.  Although we report `load_balancer_hostname` in the console, but this cannot be used to launch the product.

Reply:
Thanks, all good 👍